database/init/label.sql

```diff
@@ -0,0 +1,19 @@
+USE `labby`;
+
+DROP TABLE IF EXISTS `label`;
+DROP PROCEDURE IF EXISTS `createLabel`;
+
+DELIMITER $$
+
+CREATE PROCEDURE `createLabel` ()
+BEGIN
+CREATE TABLE `label` (
+	label_name VARCHAR(50) NOT NULL,
```

Comment:
We'll likely want to give each label an unique ID as well

Reply:
Would the PK be label_name or label_id in this case? I was initially thinking of having the label_name be the PK so that there wouldn't be an issue with deletion

Reply:
Sure I will add the label_id and make it the PK, should label_name be UNIQUE too?

Reply:
label_name doesn't have to be unique but label_id should be.